.eslintrc.js

```diff
@@ -12,7 +15,7 @@ module.exports = {
         "module": true,
         "describe": true,
         "it": true
-    },
+    },    
```

Comment:
remove trailing spaces here please (too bad the linter doesn't lint the linter file haha)